smriprep/interfaces/surf.py

```diff
@@ -42,7 +42,7 @@
 
 class _NormalizeSurfInputSpec(BaseInterfaceInputSpec):
     in_file = File(mandatory=True, exists=True, desc="Freesurfer-generated GIFTI file")
-    transform_file = File(exists=True, desc="FSL or LTA affine transform file")
+    transform_file = File(desc="FSL or LTA affine transform file")
```

Comment:
Why this change? Seems fine to have a sanity check to ensure the file actually exists

Reply:
Ah, right. I was thinking I was making it non-mandatory.

Reply:
```suggestion
    transform_file = File(exists=True, desc="FSL or LTA affine transform file")
```